Makefile

```diff
@@ -7,21 +7,23 @@ import:
 	psql --single-transaction --dbname pianoforte --file tmp/boundary.sql
 	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/city.sql data/city.csv -select name,'name:en','name:fr','name:ar',capital,type -nln city -oo X_POSSIBLE_NAMES=Lon* -oo Y_POSSIBLE_NAMES=Lat* -oo KEEP_GEOM_COLUMNS=NO -a_srs EPSG:4326
 	# Remove transaction management, as it does not cover the DROP TABLE; we'll cover the transaction manually with "psql --single-transaction"
-	sed --in-place '/BEGIN;/d' tmp/city.sql
-	sed --in-place '/END;/d' tmp/city.sql
-	sed --in-place '/COMMIT;/d' tmp/city.sql
+	sed -i.bak '/BEGIN;/d' tmp/city.sql
+	sed -i.bak '/END;/d' tmp/city.sql
+	sed -i.bak '/COMMIT;/d' tmp/city.sql
+	rm tmp/city.sql.bak
 	psql --single-transaction --dbname pianoforte --file tmp/city.sql
 	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/country.sql data/country.csv -select name,'name:en','name:fr','name:ar' -nln country -oo X_POSSIBLE_NAMES=Lon* -oo Y_POSSIBLE_NAMES=Lat* -oo KEEP_GEOM_COLUMNS=NO -a_srs EPSG:4326
 	# Remove transaction management, as it does not cover the DROP TABLE; we'll cover the transaction manually with "psql --single-transaction"
-	sed --in-place '/BEGIN;/d' tmp/country.sql
-	sed --in-place '/END;/d' tmp/country.sql
-	sed --in-place '/COMMIT;/d' tmp/country.sql
+	sed -i.bak '/BEGIN;/d' tmp/country.sql
+	sed -i.bak '/END;/d' tmp/country.sql
+	sed -i.bak '/COMMIT;/d' tmp/country.sql
+	rm tmp/country.sql.bak
 	psql --single-transaction --dbname pianoforte --file tmp/country.sql
 update:
 	env PGHOST=/var/run/postgresql/ imposm3 run -config imposm.conf
 boundary:
 	python scripts/make_boundaries.py process
-	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/boundary.sql data/boundary.json -select name,'name:en','name:fr','name:ar','name:es','name:de','name:ru' -nln itl_boundary
+	ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/boundary.sql data/boundary.json -sql 'SELECT name,"name:en","name:fr","name:ar","name:es","name:de","name:ru","ISO3166-1:alpha2" AS iso FROM OGRGeoJSON' -nln itl_boundary
```

Comment:
Humm, which ogr2ogr version are you running?
Table is name "boundary" for me, not "OGRGeoJSON".

```
ogr2ogr --config PG_USE_COPY YES -lco GEOMETRY_NAME=geometry -lco DROP_TABLE=IF_EXISTS -f PGDump tmp/boundary.sql data/boundary.json -sql 'SELECT name,"name:en","name:fr","name:ar","name:es","name:de","name:ru","ISO3166-1:alpha2" AS iso FROM OGRGeoJSON' -nln itl_boundary
ERROR 1: SELECT from table OGRGeoJSON failed, no such table/featureclass.
```

Reply:
```
$ ogr2ogr --version
GDAL 2.1.3, released 2017/20/01
```

I tried with that, but didn't work and looked at tutorials.

```
$ ogrinfo data/boundary.json
INFO: Open of `data/boundary.json'
      using driver `GeoJSON' successful.
1: OGRGeoJSON
```

Reply:
Ah, from http://www.gdal.org/drv_geojson.html

> Starting with GDAL 2.2, the layer name is built with the following logic:

>    If a "name" member is found at the FeatureCollection level, it is used.
>    Otherwise if the filename is regular (ie not a URL with query parameters), then the filename without extension and path is used as the layer name.
>    Otherwise OGRGeoJSON is used.

Can you upgrade so we align on latest? Or should I downgrade? :)

Reply:
To avoid this, we may add a name attribute to the FeatureCollection ?

https://github.com/tilery/pianoforte/blob/master/scripts/make_boundaries.py#L185

Reply:
For the record:

```
$ ogrinfo data/boundary.json
INFO: Open of `data/boundary.json'
      using driver `GeoJSON' successful.
1: boundary
```

Reply:
> To avoid this, we may add a name attribute to the FeatureCollection ?

Will this work with GDAL < 2.2?

Reply:
(Can you try? :) )

Reply:
here : http://www.gdal.org/drv_geojson.html

Starting with GDAL 2.2, the layer name is built with the following logic:

* If a "name" member is found at the FeatureCollection level, it is used.
* Otherwise if the filename is regular (ie not a URL with query parameters), then the filename without extension and path is used as the layer name.
* Otherwise OGRGeoJSON is used.

Reply:
No, unforunately doesn't work.

I will try to upgrade

Reply:
Ok, Gdal 2.2.x needed cf : #21